chemclipse/plugins/org.eclipse.chemclipse.ux.extension.xxd.ui/src/org/eclipse/chemclipse/ux/extension/xxd/ui/swt/ExtendedComparisonScanUI.java

```diff
@@ -359,8 +359,8 @@ private void updateScanComparisonNormal() {
 			int molWeightUnknown = getMolWeight(textWeightUnknownControl);
 			int molWeightReference = getMolWeight(textWeightReferenceControl);
 			BaseChart baseChart = scanChartUI.getBaseChart();
-			baseChart.shiftSeries("Scan -- scan1", IExtendedChart.X_AXIS, -molWeightUnknown);
-			baseChart.shiftSeries("Scan -- scan2", IExtendedChart.X_AXIS, -molWeightReference);
+			baseChart.shiftSeries("Unknown Scan", IExtendedChart.X_AXIS, -molWeightUnknown);
```

Comment:
That breaks displaying the hybrid search delta m/z view, as creating the series ID in ScanChartSupport.java has been changed. The series ID has been changed to "Unkown" and "Reference".